### PR TITLE
CVE-2011-3022 and CVE-2015-1245

### PR DESCRIPTION
**Pull request by johannvoelkening**
Committing assigned CVE's

### CODE REVIEW

File: cves/CVE-2011-3022.yml [hunk 1]
```diff
@@ -15,7 +15,14 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description:
+  Javascript used by Chrome to translate web pages is downloaded over HTTP.
+  This script is executed in the context of the web page to be translated.
+   
+  translate/translate_manager.cc in Google Chrome before 17.0.963.56 and 19.x
```
Comment (jonobrien) [inline cves/CVE-2011-3022.yml]: The instructions say to remove version numbers

File: cves/CVE-2011-3022.yml [hunk 2]
```diff
@@ -85,7 +94,7 @@ subsystem:
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
   answer: 
-  name: 
+  name: UI and chrome/browser/translate subsystem
```
Comment (jonobrien) [inline cves/CVE-2011-3022.yml]: You need to fill in the section above this about who discovered the vulnerability and when it was discovered

Comment (jonobrien) [inline cves/CVE-2011-3022.yml]: section below this about interesting commits as well lines ~105ish should be completed

File: cves/CVE-2011-3022.yml [hunk 3]
```diff
@@ -85,7 +94,7 @@ subsystem:
...
 interesting_commits:
   question: |
     Are there any interesting commits between your VCC(s) and fix(es)?
```
Comment (jonobrien) [inline cves/CVE-2011-3022.yml]: you should fill in this section

File: cves/CVE-2015-1245.yml [hunk 4]
```diff
@@ -15,7 +15,17 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  Use-after-free vulnerability in the OpenPDFInReaderView::Update function in 
+  browser/ui/views/location_bar/open_pdf_in_reader_view.cc in Google Chrome
```
Comment (jonobrien) [inline cves/CVE-2015-1245.yml]: "Remove references to versions, specific filenames..."

File: cves/CVE-2015-1245.yml [hunk 5]
```diff
@@ -15,7 +15,17 @@ description_instructions: |
...
+  Use-after-free vulnerability in the OpenPDFInReaderView::Update function in 
+  browser/ui/views/location_bar/open_pdf_in_reader_view.cc in Google Chrome 
+  before 41.0.2272.76 might allow user-assisted remote attackers to cause a 
+  denial of service (heap memory corruption) or possibly have unspecified other 
+  impact by triggering interaction with a PDFium "Open PDF in Reader" button that 
+  has an invalid tab association. 
+  
+  This can allow a malicious web site to cause a user to be redirected to a different
+  page when opening a link on the original page with the "Open PDF in Reader" button.
+  This can cause a user's information to be stolen.
```
Comment (jonobrien) [inline cves/CVE-2015-1245.yml]: Is this XSS and heap overflow and use after free? Can you please elaborate more?

File: cves/CVE-2015-1245.yml [hunk 6]
```diff
@@ -84,7 +94,7 @@ subsystem:
     directory names. Look at comments in the code. Look at the bug reports how
```
Comment (jonobrien) [inline cves/CVE-2015-1245.yml]: you should finish the above section

Comment (jonobrien) [inline cves/CVE-2015-1245.yml]: Also the unit test section above that has not been touched.

File: cves/CVE-2015-1245.yml [hunk 7]
```diff
@@ -84,7 +94,7 @@ subsystem:
...
     the bug report was tagged.
   answer: 
-  name: 
+  name: UI subsystem
 interesting_commits:
   question: |
     Are there any interesting commits between your VCC(s) and fix(es)?
```
Comment (jonobrien) [inline cves/CVE-2015-1245.yml]: Are there no interesting commits?

Comment (jonobrien) [inline cves/CVE-2015-1245.yml]: Also everything below here is not done either.

Comment (jonobrien) [inline cves/CVE-2015-1245.yml]: Should be completed.

File: cves/CVE-2011-3022.yml (hunk 1 above)
Comment (jonobrien) [inline cves/CVE-2011-3022.yml]: CWE should be filled in, line 3

File: cves/CVE-2015-1245.yml (hunk 4 above)
Comment (jonobrien) [inline cves/CVE-2015-1245.yml]: line 3 CWE should be filled in.

File: cves/CVE-2015-1245.yml (hunk 7 above)
Comment (smcintosh881) [inline cves/CVE-2015-1245.yml]: There seems to be missing parts and a little sparse on details. Where this CVE's poorly documented?

File: cves/CVE-2011-3022.yml [hunk 8]
```diff
@@ -15,7 +15,14 @@ description_instructions: |
...
-description: 
+description:
+  Javascript used by Chrome to translate web pages is downloaded over HTTP.
+  This script is executed in the context of the web page to be translated.
+   
+  translate/translate_manager.cc in Google Chrome before 17.0.963.56 and 19.x 
+  before 19.0.1036.7 uses an HTTP session to exchange data for translation, 
+  which allows remote attackers to obtain sensitive information by sniffing 
+  the network.
 bounty:
```
Comment (smcintosh881) [inline cves/CVE-2011-3022.yml]: Missing bounty amount and who/how the bug was discovered.

**Pull request merged**